src/Data/Nat/Properties.agda

```diff
@@ -1476,6 +1480,15 @@ pred[m∸n]≡m∸[1+n] (suc m) (suc n) = pred[m∸n]≡m∸[1+n] m n
 ------------------------------------------------------------------------
 -- Properties of _∸_ and _≤_/_<_
 
+m+n≤p⇒m≤p∸n : ∀ m n p → m + n ≤ p → m ≤ p ∸ n
```

Comment:
Very minor but the style guide dictates that `o` should be used as the third name for a natural number rather than `p`.

Reply:
:+1: